SR2025/2024-09-18-secondary-contact-info.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2025 secondary contacts, CC primary contacts
```

Comment:
Hrm, just spotted that Gmail's mail merge doesn't support CC :( Not sure how best to do this. Might need to script something ourselves.

Reply:
Have scripted this and sent it out. Added the script here in case it's useful in future.